packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx

```diff
@@ -20,10 +21,16 @@ const ChevronContainer = styled.div`
     width: 30px;
 `;
 
-const ChevronIcon = styled(Icon)`
+const ChevronIcon = styled(Icon)<{ isActive: boolean; canAnimate: boolean }>`
     padding: ${spacingsPx.sm};
     border-radius: 50%;
     transition: background 0.2s;
+
+    ${({ canAnimate, isActive }) =>
+        canAnimate &&
+        css`
+            animation: ${isActive ? ROTATE_180_UP : ROTATE_180_DOWN} 0.2s linear 1 forwards;
```

Comment:
honestly, I believe this can be handled by a simple `transition`, don't u think? 🤔 `animation` sounds like something that's happening either on mount, perpetually or when there's orchestration involved, e.g. multiple properties are being changed. Here we are simply changing one `transform`.

Reply:
changing this to a transition doesn't work...

Reply:
https://github.com/trezor/trezor-suite/pull/10754#discussion_r1467882919

Reply:
281bbd1074b0682bc3fbacbdbc0e08ed92c67915

Actually I didn't keep any animation since for example similar the chevron for show and hide security section on dashboard doesn't have any animation at all